src/Lesson1/Task1/ArrayExcerciseVariantOne.java

```diff
@@ -0,0 +1,83 @@
+package Lesson1.Task1;
+
+/*
+* 1 Написать программу, осуществляющую обработку массива. Исходные условия:
+массив содержит только целые числа от -10 до 10 (допускается генерация
+элементов с помощью метода (int) Math.random()), программа должна выводить
+в консоль исходный массив и полученный результат, количество элементов в
+массиве 20.
+Варианты задания:
+1. В массиве целых чисел поменять местами максимальный отрицательный
+элемент и минимальный положительный.*/
+
+/**
+ * @author Andrii Prokofiev
+ * @version 1.0
+ */
+
+public class ArrayExcerciseVariantOne {
+
+    public static void main(String[] args) {
+        ArrayExcerciseVariantOne arrayExcercises = new ArrayExcerciseVariantOne();
+        arrayExcercises.startProgram();
+    }
+
+    public void startProgram() {
+        int[] arr = getIntArray();
+        int minNegativeIndex = getMaxNegativeIndex(arr);
+        int minPositiveIndex = getMinPositive(arr);
+        System.out.println("Index with min neg = " + minNegativeIndex);
+        System.out.println("Index with min pos = " + minPositiveIndex);
```

Comment:
It's better to print log information right inside of method, where is this logic done, so move line 29 and 30 to specific method.